package.json

```diff
@@ -168,6 +169,7 @@
 		"oauth-1.0a": "^2.2.6",
 		"papaparse": "^5.1.1",
 		"passport": "^0.4.1",
+		"passport-custom": "^1.1.1",
```

Comment:
Is this in use? passport-custom

Reply:
Yes, this is the base of the ldap strategy. The passport-ldap lib only supports only one LDAP configuration setup in the constructor. That does not work for our multiple LDAPs.
Same will be the case for OAuth.